src/main/java/cloud/eppo/ufc/dto/DiscriminableAttributes.java

```diff
@@ -0,0 +1,10 @@
+package cloud.eppo.ufc.dto;
+
+public interface DiscriminableAttributes {
```

Comment:
Created this interface as some parts of bandit evaluation don't care about numeric vs. categorical (i.e., _variation_ assignment) while others do (i.e., _action_ assignment). This interface:
1. Allows upstream developers to easily pass in explicitly separated attributes or not as they please.
2. Simplifies the bandit code that interacts with the attributes